packages/lesswrong/lib/vulcan-lib/routes.ts

```diff
@@ -42,6 +42,7 @@ export type Route = {
   disableAutoRefresh?: boolean,
   initialScroll?: "top"|"bottom",
   standalone?: boolean // if true, this page has no header / intercom
+  staticHeader?: boolean // if true, the page header is not sticky to the top of the screen
```

Comment:
I added this because the table headers on the Edit Digest page are sticky, and I didn't want to fight with the page header